tensorrt_llm/_torch/pyexecutor/py_executor.py

```diff
@@ -918,8 +918,7 @@ def _executor_loop(self):
 
                 self._pad_attention_dp_dummy_request()
 
-                if self.draft_model_engine is not None or hasattr(
-                        self, 'drafter') and self.drafter is not None:
+                if hasattr(self, 'drafter') and self.drafter is not None:
```

Comment:
Looks like the `hasattr(self, 'drafter')` part is not necessary (it's always true)